utils/python/CIME/SystemTests/lii.py

```diff
@@ -43,49 +43,38 @@ def build(self, sharedlib_only=False, model_only=False):
             for filename in glob.glob(r'user_nl_clm*'):
```

Comment:
The lii test is very specific to the clm model and compares an initial run from internally interpolated initial data to one from externally interpolated initial data.